python/cugraph/cugraph/link_prediction/woverlap.py

```diff
@@ -66,14 +65,19 @@ def overlap_w(input_graph, weights, vertex_pair=None):
     >>>                   dtype=['int32', 'int32', 'float32'], header=None)
     >>> G = cugraph.Graph()
     >>> G.from_cudf_edgelist(M, source='0', destination='1')
-    >>> df = cugraph.overlap_w(G, M[2])
+    >>> weights = cudf.DataFrame()
```

Comment:
This example got more complicated, so you might add a comment or two if you can keep them brief.